### PR TITLE
Do not show 0s for cooldown label

### LINKED ISSUE #1493: [Auth UI] Should not show 0s for resend count down
Upon reaching 0s, the resend verification code countdown will just stay as 'Resend (0s)' but not 'Resend'.
- Skygear Server Date/Version: Not sure
- Application Platform: Tested on Rees

#### Expected Results
Should be just 'Resend' when reached 0s

#### Actual Results
It shows Resend (0s)' but not 'Resend'

#### Steps to reproduce
1. Open signup / login page
2. Select phone number / email to reach verification page
3. Wait until the count down to become 0s

#### Screenshot
image

### PR DESCRIPTION
**Pull request by louischan-oursky**
fix #1493

### CODE REVIEW

File: static/authui/js/main.js [hunk 1]
```diff
@@ -158,9 +160,14 @@ window.addEventListener("load", function() {
...
-      el.textContent = (el.textContent || "").replace(/\d+/, String(displaySeconds));
+      if (displaySeconds === 0) {
+        el.textContent = label;
+      } else {
+        el.textContent = labelUnit.replace("%d", String(displaySeconds));
+      }
     }
-    token = setInterval(tick, 500);
+    // 24fps => 1 / 24 * 1000 = 42ms
+    token = setInterval(tick, 42);
```
Comment (kiootic) [inline static/authui/js/main.js:170]: Actually, why do we need to use smaller interval?

Comment (louischan-oursky) [reply static/authui/js/main.js:170]: 0.5s sometimes does not look smooth.

Comment (kiootic) [reply static/authui/js/main.js:170]: Let's use `requestAnimationFrame` instead?

Comment (louischan-oursky) [reply static/authui/js/main.js:170]: `requestAnimationFrame` is a bit over kill.

Comment (kiootic) [reply static/authui/js/main.js:170]: I don't think it's overkill. The issue we have at hand is 'countdown animation is not smooth', so using a proper animation API to update it is quite appropriate.

**Pull request merged**